website/content/en/docs/ansible/development-tips.md

```diff
@@ -379,29 +412,25 @@ status:
     type: Running
 ```
 
-Ansible Operator also allows you as the developer to supply custom
-status values with the `k8s_status` Ansible Module, which is included in
-[operator_sdk util collection](https://galaxy.ansible.com/operator_sdk/util).
-
-This allows the developer to update the `status` from within Ansible
-with any key/value pair as desired. By default, Ansible Operator will
-always include the generic Ansible run output as shown above. If you
-would prefer your application *not* update the status with Ansible
-output and would prefer to track the status manually from your
-application, then simply update the watches file with `manageStatus`:
+An Ansible Operator also allows you to supply custom status values with the
+`k8s_status` Ansible module, which is included in
+[operator_sdk.util](https://galaxy.ansible.com/operator_sdk/util) collection.
+You can update the `status` from within Ansible with any key/value pairs as
+desired. If you want to track the CR status manually from your application,
+you can update the `watches.yaml` file with `manageStatus`, as shown below:
```

Comment:
Could you revert to the previous description which is more accurate?

Reply:
Actually I didn't add or remove any text. A summary of the changes made to the paragraph:

- Changed "Ansible Operator" to "An Ansible Operator" to make it generic and grammatically correct.
- Simplified "you as the developer" to "you" because we are in a doc called development tips after all.
- Dropped "By default, Ansible Operator will always include the generic Ansible run output as shown above." because we already said that in previous paragraph.

This only missing piece (after rereading this several times, :)) is `If you would prefer your application *not* update the status with Ansible output`, which I'll add back in next commit.